probeinfo_api.yaml

```diff
@@ -481,7 +483,17 @@ components:
 
     Description:
       type: string
-      description: A brief free-text description of the repository
+      description: A brief free-text description of this application variant.
+
+    CanonicalAppDescription:
+      type: string
+      description: |
+        A brief free-text description of an application, applicable across any variants.
+
+        This should not be materialized in ETL, but rather only made available in user-facing views since it may be verbose
+        and it may change over time.
```

Comment:
Are these flipped? Something that changes over time and is used in the user-facing view sounds like `Description`. A canonical description sounds like it should be as static, and terse, as possible.

Reply:
The "canon" as to what an app is called and how we describe it may change over time. This field is intended to always reflect the current state of how we think about the application.

In contrast, the `app_name` is a value that must stay stable over time because it's used as a technical hook for filtering.

That said, the `canonical` may not be necessary here. `app_description` would be sufficient to disambiguate from the channel-level `description`. I introduced `canonical_app_name` as a complement to the technical `app_name`. Perhaps it would be obvious to users that a description field is not something that should be materialized in ETL.

Reply:
I like the sound of just using `app_description` everywhere. I'll give that a try.

Reply:
Thanks for clarifying, @jklukas. That makes sense now.